README.md

```diff
@@ -1,2 +1,86 @@
-# preflight
-Automatically perform Kubernetes cluster configuration checks using Open Policy Agent (OPA)
+# Jetstack Preflight
+
+Preflight is a tool to automatically perform Kubernetes cluster configuration checks using [Open Policy Agent (OPA)](https://www.openpolicyagent.org/).
+
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+**Table of Contents**
+
+- [Jetstack Preflight](#jetstack-preflight)
+    - [Background](#background)
+    - [Preflight Packages](#preflight-packages)
+    - [Install Preflight](#install-preflight)
+    - [Use Preflight locally](#use-preflight-locally)
+    - [Get periodic reports by running Preflight as a CronJob](#get-periodic-reports-by-running-preflight-as-a-cronjob)
+
+<!-- markdown-toc end -->
+
+
+## Background
+
+Preflight was originally designed to automate Jetstack's
+production readiness assessments.
+These are consulting sessions in which a Jetstack engineer inspects a customer's
+cluster to suggest improvements and identify configuration issues. 
+The product of this assessment is a report
+which describes any problems and offers remediation advice.
+
+While these assessments have provided a lot of value to many customers,
+with a complex system like Kubernetes its hard to thoroughly check everything.
+Automating the checks allows them to be more comprehensive and much faster.
+
+The automation also allows the checks to be run repeatedly,
+meaning they can be deployed in-cluster to provide continuous configuration checking.
+
+This enables new interesting use cases as policy compliance audits.
+
+Jetstack plans to offer Preflight as a subscription service for customers,
+and to assist Customer Reliability Engineers (CREs) in keeping clusters running smoothly.
```

Comment:
I don’t think the wording here is right - preflight is open source, what you describe here is something else (some of the rule packs), and also partially TBD

Reply:
Ok, deleting this whole paragraph to avoid confusion.